### DISCUSSION TITLE
Are async calculations allowed?

### DISCUSSION THREAD

**Discussion by Jakabi107** [SebLague/Chess-Challenge#184] [General]
I haven't tried it yet, but would it be cheating using an async method to calculate future positions while the other player is thinking?

**Comment by penguinsrepic** [comment]
No,
"As implied by the allowed namespaces, you may not read data from a file or access the internet, nor may you create any new threads or tasks to run code in parallel/in the background." (from the readme)

**Reply by Jakabi107** [reply]
Oh - must have overread it. Thanks a lot!!
